include/h2o.h

```diff
@@ -1234,6 +1234,10 @@ struct st_h2o_req_t {
      * list of request headers
      */
     h2o_headers_t headers;
+    /**
+     * list of request trailers
```

Comment:
```suggestion
     * list of request trailers; `trailers.entries` is non-NULL only when the list is non-empty
```
Please see my other comment for the rationale.